bundles/binding/org.openhab.binding.mqtt/README.md

```diff
@@ -28,7 +28,7 @@ In order to consume or publish messages to an MQTT broker, you need to define al
 | Property             | Default | Required | Description                                                 |
 |----------------------|---------|:--------:|-------------------------------------------------------------|
 | `<broker>`.url       |         | Yes      | URL to the MQTT broker, e.g. tcp://localhost:1883 or ssl://localhost:8883 |
-| `<broker>`.clientId  |         | Yes/No   | Client ID to use when connecting to the broker. If not provided a random default is generated. Usually restricted to 23 characters in length. (see the `allowLongerClientIds` setting) |
+| `<broker>`.clientId  | random  | Yes*     | Client ID to use when connecting to the broker. If not provided a random default is generated. Usually restricted to 23 characters in length. (see the `allowLongerClientIds` setting) |

```

Comment:
What does Yes* mean?  With a "random default is generated," I would think No* would be more appropriate, but with "*" given below the table to explain.  Marking something with an asterisk but not explaining below is one of my pet peeves! 😄